packages/components/psammead-media-indicator/src/mediaIcons.jsx

```diff
@@ -9,17 +9,27 @@ const MediaIcon = styled.svg`
   fill: ${C_EBON};
 `;
 
+const VideoMediaIcon = styled(MediaIcon)`
+  width: 0.75rem;
+  height: 0.75rem;
+`;
+
+const AudioMediaIcon = styled(MediaIcon)`
```

Comment:
I have a feeling the reason we did html attributes for width and height before, not CSS, was to ensure the icons dont go huge when css is disabled. This way, _i think_ they go full screen with you disable css. Havent checked though

Reply:
True, so then we will have to accept that they don't scale when the text is resided or find another workaround, because as far as I know, we can't put `rem` directly in the html attributes

Reply:
Could we have both? width attribute for no css, width css for with css?

Reply:
That worked 💅🏼